### PR TITLE
Improve localization strings

### PR DESCRIPTION
**Pull request by xavivars**
Hi,

It'd be very benefitial if strings were not sliced into pieces in order to add HTML tags in between, like

https://github.com/rtMediaWP/rtMedia/blob/develop/app/importers/RTMediaMigration.php#L381

This makes create good translations almost impossible.

Also, giving context to what the replacement strings do helps a lot.

### CODE REVIEW

File: app/importers/RTMediaMigration.php [hunk 1]
```diff
@@ -378,11 +378,15 @@ function test() {
...
-				<p> <?php esc_html_e( 'Please Backup your ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'DATABASE', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' and ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'UPLOAD', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' folder before Migration.', 'buddypress-media' ); ?>
+				<p> 
+					<?php 
+			/* translators: %1$s gets replaced by '<strong>', and %2$s by '</strong>' */
+			echo sprintf(
+				__('Please Backup your %1$sDATABASE%2$s and %1$sUPLOAD%2$s folder before Migration.', 'buddypress-media' )
```
Comment (xavivars) [inline app/importers/RTMediaMigration.php]: Sorry about hte Syntax error. Regarding the WP Coding Standars, I already tried to follow them (i.e. https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#opening-and-closing-php-tags) but probably missed something.

File: app/importers/RTMediaMigration.php [hunk 2]
```diff
@@ -378,11 +378,14 @@ function test() {
...
 			<div class="error">
-				<p> <?php esc_html_e( 'Please Backup your ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'DATABASE', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' and ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'UPLOAD', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' folder before Migration.', 'buddypress-media' ); ?>
+				<p> 
+					<?php 
+			/* translators: %1$s gets replaced by '<strong>', and %2$s by '</strong>' */
+			echo sprintf(
```
Comment (pradeep910) [inline app/importers/RTMediaMigration.php]: @BhargavBhandari90 Fix the indentation.

**Comment by BhargavBhandari90** [comment]
@pradeep910 requested change has been done now. Please check now.

**Comment by xavivars** [comment]
This is a single message, but there are plenty of them in the plugin, so (ideally) IMHO all messages should be changed to follow this pattern instead.

**Comment by pradeep910** [comment]
@xavivars True. Thanks for pointing it out. 

@BhargavBhandari90 Please do the same thing at other places, where we have multiple translation function calls.

**Comment by BhargavBhandari90** [comment]
Will continue in https://github.com/rtMediaWP/rtMedia/pull/990

**Pull request closed without merging**